setup.py

```diff
@@ -26,6 +26,8 @@
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     packages=["pyramid_heroku"],
+    package_data={'pyramid_heroku': ['py.typed']},
```

Comment:
This one is not needed, since we have MANIFEST.in.

That said `MANIFEST.in` is probably missing `graft pyramid_heroku`

Reply:
Making changes.
MANIFEST.in does not need `py.typed` right ?